### DISCUSSION TITLE
Do Screenshots of every app/every 5min

### DISCUSSION THREAD

**Discussion by letelba** [ActivityWatch/activitywatch#1096] [Ideas]
Would love to get more granullar of what I'm doing, so If I could see what I'm doing through regular screenshots, that would be awesome!

**Comment by monkrock** [comment]
🙏 Would love to have that feature. I am doing a lot of CAD modelling and based on a screenshot I would be able to list exactly what parts I was working on. It's the only thing holding me back from finally switching from ManicTime.
